packages/turf-simplify/index.js

```diff
@@ -147,9 +147,17 @@ function simpleFeature(geom, properties) {
 
 function simplifyLine(coordinates, tolerance, highQuality) {
     return simplify(coordinates.map(function (coord) {
-        return {x: coord[0], y: coord[1]};
+        if (coord.len === 2){
+            return {x: coord[0], y: coord[1]};
+        } else {
+            return {x: coord[0], y: coord[1], z: coord[2]};
+        }
     }), tolerance, highQuality).map(function (coords) {
-        return [coords.x, coords.y];
+        if (coords.len === 2){
```

Comment:
Sorry, this code doesn't work: if you run tests, it writes new files with `null` altitude values. The `length` check wouldn't work, because Objects don't have length.